public/controllers/management/components/management/configuration/office365/components/api-auth-tab/api-auth-tab.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Wazuh app - React component ApiAuthTab
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+import React from 'react';
+import WzConfigurationSettingsTabSelector from '../../../util-components/configuration-settings-tab-selector';
+import WzConfigurationSettingsGroup from '../../../util-components/configuration-settings-group';
+import { HELP_LINKS, OFFICE_365 } from '../../constants';
+
+export type ApiAuthProps = {
+  agent: { id: string };
+  wodleConfiguration: any;
+};
+
+export const ApiAuthTab = ({ agent, wodleConfiguration }: ApiAuthProps) => {
+  const columns = [
+    { field: 'tenant_id', label: 'Tenant Id' },
+    { field: 'client_id', label: 'Client Id' },
+    { field: 'client_secret', label: 'Client Secret' },
+  ];
```

Comment:
**nitpick**: the `client_secret_path` setting is missing. We can get some information about the setting, here: https://github.com/wazuh/wazuh-documentation/blob/develop/source/user-manual/reference/ossec-conf/office365-module.rst#api_authclient_secret_path